pkgs/firehose/lib/src/changelog.dart

```diff
@@ -4,69 +4,64 @@
 
 import 'dart:io';
 
-import 'package:collection/collection.dart';
-
 class Changelog {
```

Comment:
This entire class can be optimized to not read the file from scratch on every operation.
Unless it's deliberately doing it to avoid using stale data, it probably should cache the `List<String>` lines and/or `List<_Section>` sections.

Reply:
I suspect that worrying about stale data was not a use-case here; some simple caching seems reasonable.